build.gradle

```diff
@@ -54,8 +62,15 @@ afterEvaluate {
 }
 //endregion
 
+String kgpVersion = KotlinPluginWrapperKt.getKotlinPluginVersion(project)
+logger.info("Using Kotlin ${KotlinPluginWrapperKt.getKotlinPluginVersion(it)}")
```

Comment:
I'm getting `> Could not get unknown property 'it' for root project 'kotlinx-benchmark' of type org.gradle.api.Project.` locally.

Reply:
oops, bad copy/paste. I pushed a fix.